web/controllers/installation_controller.ex

```diff
@@ -2,8 +2,11 @@ defmodule Dwylbot.InstallationController do
   use Dwylbot.Web, :controller
 
   plug :authenticate_user when action in [:index]
+  @github_api Application.get_env(:dwylbot, :github_api)
 
   def index(conn, _params) do
-    render conn, "index.html"
+    token = conn.assigns.current_user.token
```

Comment:
yes, the authentication plug on top of the file ([here](https://github.com/dwyl/dwylbot/pull/54/files#diff-ba9b85e7522d1e17da4c06e6cb5704c9L4)) makes sure that the index action is only accessible is the user is loggedin ie if conn.assigns.current_user is defined